flow/designs/intel22/aes/config.mk

```diff
@@ -15,6 +15,6 @@ export DIE_AREA    = 0 0 250 250
 export CORE_AREA   = 1.26 1.89 248 248
 
 
-export PLACE_DENSITY = uniform
+export PLACE_DENSITY = 0.35 
```

Comment:
Why change this parameter?

Reply:
I was trying to run all the test cases using DREAMPlace. DREAMPlace does not support place density as 'uniform' so I changed it to this value.

Reply:
Please make sure to document this somewhere. Preferably in the same place where you instruct on how to enable DREAMPlace. Also, make sure to list all current limitations if there are more.

Finally, it might be a good idea to create some kind of check for this limitation either on the Makefile or Tcl. I do not know what kind of error DREAMPlace would give, if it would be clear to the user what is the problem.